src/admin-views/rsvp-metabox-not-going.php

```diff
@@ -0,0 +1,20 @@
+<div
+	class="input_block ticket_advanced_Tribe__Tickets__RSVP tribe-dependent"
+	data-depends="#Tribe__Tickets__RSVP_radio"
+	data-condition-is-checked
+>
+	<label
+		for="Tribe__Tickets__RSVP_not_going"
```

Comment:
not crucial, seeing as this is admin views, but better if we can keep this and the corresponding id field lowercase (the class `tribe-rsvp-field-not-going` would be a great id for this)

Reply:
I copied from another admin view, just replaced a few things. Will update